README.md

```diff
@@ -16,48 +16,48 @@ Binaries, including libraries, CLIs and docker images, are licensed under the [R
 ## Installation
 
 1. Install Rust - this requires Rust 1.70+ (if rust is already installed, upgrade with `rustup update`)
```

Comment:
Ops I should revert this... cargo fmt ran when ai saves the file and it made this changes..

I did add row in table with info about being able to run just `resim show`

Reply:
It's all right. I can see no semantic change, rather than the addition of `resim show`